.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
What's this file for? If it's useful for developing from VSCode, can you make a separate PR for it?